config_examples/dblp_mapping.json

```
@@ -0,0 +1,59 @@
+{
+  "nodes": [
+    {
+      "nodeLabel": "Author",
+      "xpath": "/dblp/article/author",
+      "innerTextMappedTo": "name",
+      "mappedAttributes": {},
+      "mappedChildElementInnerTexts": {}
+    },
+    {
+      "nodeLabel": "Publication",
+      "xpath": "/dblp/article",
+      "innerTextMappedTo": null,
+      "mappedAttributes": {
+        "publtype": "type"
+      },
+      "mappedChildElementInnerTexts": {
+        "title": "title",
+        "year": "year"
+      }
+    }
+  ],
+  "edges": [
+    // krawędź nieskierowana (dziecko)---(dziecko)
+    {
+      "edgeLabel": "CollaboratedWith",
+      "edgeType": "undirected-between-children",
```

```suggestion
      "edgeType": "child-child",
```
Prościej i nadal jednoznacznie